shared/java/org/jetbrains/jwm/Window.java

```diff
@@ -232,6 +232,11 @@ public Window setMouseCursor(MouseCursor cursor) {
      */
     public abstract float getOpacity();
 
+    /**
+     * <p>check whether OS currently uses high contrast mode.</p>
+     */
+    public abstract boolean isHighContrast();
```

Comment:
You were correct in #143 that themes are per-app, not per-window. Apologies if it was me who confused and/or suggested this method to be on a Window class.

It could either go on App or on a new class called Theme, similar how Clipboard groups all the functions related to clipboard manipulation. I suggest Theme.

Note: this would be different from dark/light enum, this will be a class containing static methods like isHighContrast() etc. We’ll have to come up with something different re: dark/light

Reply:
According to your advice, I added a Theme class.